src/components/TextInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+function RenderTextField({ ...props }) {
```

Comment:
https://github.com/erikras/redux-form-material-ui/blob/master/src/TextField.js
Used a createComponent class which is a general class that does this and more 
https://github.com/erikras/redux-form-material-ui/blob/master/src/createComponent.js